### PR TITLE
Create supported-os.md for .NET 9

### PR DESCRIPTION
**Pull request by rbhanda**

### CODE REVIEW

File: release-notes/9.0/supported-os.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# .NET 9 - Supported OS versions
+
+[.NET 9](README.md) is a [Long Term Support (LTS)](../../release-policies.md) release and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```
Comment (AaronRobinsonMSFT) [inline release-notes/9.0/supported-os.md:3]: I thought even number releases were LTS only. Was this recently changed?

Comment (richlander) [reply release-notes/9.0/supported-os.md:3]: No. That's a copy/paste error.

```suggestion
[.NET 9](README.md) is a [Standard Term Support (STS)](../../release-policies.md) release and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```

File: release-notes/9.0/supported-os.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+For issues with .NET on operating systems not listed here, please open a GitHub issue in the appropriate .NET repository or contact the OS maintainer community . See [.NET Repos](../../Documentation/core-repos.md) for the repository list.
```
Comment (AaronRobinsonMSFT) [inline release-notes/9.0/supported-os.md:5]:
```suggestion
For issues with .NET on operating systems not listed here, please open a GitHub issue in the appropriate .NET repository or contact the OS maintainer community. See [.NET Repos](../../Documentation/core-repos.md) for the repository list.
```

File: release-notes/9.0/supported-os.md (hunk 1 above)
Comment (rbhanda) [reply release-notes/9.0/supported-os.md]: Updated the doc with "STS" in-line with our support policy

File: release-notes/9.0/supported-os.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+.NET 8 is not supported being run (emulated) via [QEMU](https://www.qemu.org/). QEMU is used, for example, to emulate Arm64 containers on x64, and vice versa.
+
+## Support changes from .NET 6.0
+
+The following operating systems are no longer supported, starting with .NET 7.0.
+
+OS                                    | Version                 | Architectures     |
+--------------------------------------|-------------------------|-------------------|
+[Windows Client][Windows-client]      | 7 SP1, 8.1      | x64, x86          |
```
Comment (AaronRobinsonMSFT) [inline release-notes/9.0/supported-os.md:120]: There is a bit more nuance to this if I recall. @richlander @jkotas Is this how we want to express Windows 7 support?

File: release-notes/9.0/supported-os.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+[Nano Server][Nano-Server]            | Version 1809+           | x64             | [Windows Server][Windows-Server-lifecycle]
+
+[Windows-client]: https://www.microsoft.com/windows/
+[Windows-lifecycle]: https://support.microsoft.com/help/13853/windows-lifecycle-fact-sheet
+[win-client-docker]: https://hub.docker.com/_/microsoft-windows
+[Windows-Server-lifecycle]: https://learn.microsoft.com/windows-server/get-started/windows-server-release-info
+[Nano-Server]: https://learn.microsoft.com/windows-server/get-started/getting-started-with-nano-server
+[Windows-Server]: https://learn.microsoft.com/windows-server/
+
+.NET 8 is supported in the x64 emulator on Windows 11 Arm64.
```
Comment (bartonjs) [inline release-notes/9.0/supported-os.md:24]: 9, presumably?

File: release-notes/9.0/supported-os.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+[glibc]: https://www.gnu.org/software/libc/
+[musl]: https://musl.libc.org/
+
+## macOS
+
+OS                            | Version                   | Architectures     |
+------------------------------|---------------------------|-------------------|
+[macOS][macOS]                | 12.0+                     | x64, Arm64        |
+
+.NET 8 is supported in the Rosetta 2 x64 emulator.
```
Comment (bartonjs) [inline release-notes/9.0/supported-os.md:85]: 9?

File: release-notes/9.0/supported-os.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+[iOS]: https://support.apple.com/ios
+[tvOS]: https://support.apple.com/apple-tv
+
+## QEMU
+
+.NET 8 is not supported being run (emulated) via [QEMU](https://www.qemu.org/). QEMU is used, for example, to emulate Arm64 containers on x64, and vice versa.
+
+## Support changes from .NET 6.0
+
+The following operating systems are no longer supported, starting with .NET 7.0.
```
Comment (bartonjs) [inline release-notes/9.0/supported-os.md:116]: The section says 6, the words say 7; I suspect they both mean 8?

File: release-notes/9.0/supported-os.md (hunk 4 above)
Comment (richlander) [reply release-notes/9.0/supported-os.md:24]:
```suggestion
.NET 9 is supported in the x64 emulator on Windows 11 Arm64.
```

File: release-notes/9.0/supported-os.md (hunk 5 above)
Comment (richlander) [reply release-notes/9.0/supported-os.md:85]:
```suggestion
.NET 9 is supported in the Rosetta 2 x64 emulator.
```

File: release-notes/9.0/supported-os.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+[tvOS][tvOS]                  | 12.0+                   | Arm64             |
+[tvOS Simulator][tvOS]        | 12.0+                   | x64, Arm64        |
+[MacCatalyst][macOS]          | 12.0+                   | x64, Arm64        |
+
+[iOS]: https://support.apple.com/ios
+[tvOS]: https://support.apple.com/apple-tv
+
+## QEMU
+
+.NET 8 is not supported being run (emulated) via [QEMU](https://www.qemu.org/). QEMU is used, for example, to emulate Arm64 containers on x64, and vice versa.
```
[the remark this answers is not in the document]
Comment (richlander) [reply release-notes/9.0/supported-os.md:112]:
```suggestion
.NET 9 is not supported being run (emulated) via [QEMU](https://www.qemu.org/). QEMU is used, for example, to emulate Arm64 containers on x64, and vice versa.
```

File: release-notes/9.0/supported-os.md (hunk 3 above)
Comment (richlander) [inline release-notes/9.0/supported-os.md:120]: I don't think we need this content. Also, the references to .NET 6 and 7 are confusing.

```suggestion
```

Comment (richlander) [reply release-notes/9.0/supported-os.md:120]: I added a suggestion to remove that.

File: release-notes/9.0/supported-os.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
+# .NET 9 - Supported OS versions
+
+[.NET 9](README.md) is a [Standard Term Support (STS)](../../release-policies.md) release and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```
Comment (richlander) [inline release-notes/9.0/supported-os.md:3]:
```suggestion
[.NET 9](README.md) is a [Standard Term Support (STS)](../../release-policies.md) release and [is supported](../../support.md) on multiple operating systems per their lifecycle policy.
```

**Review by richlander — APPROVED** [review]

**Comment by richlander** [comment]
Looks like we need a `README.md` file. We should create a largely blank one.

File: release-notes/9.0/supported-os.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+[Oracle Linux][Oracle-Linux]          | 8+                    | x64               | [Oracle][Oracle-lifecycle]
+[Red Hat Enterprise Linux][RHEL]      | 8+                    | x64, Arm64        | [Red Hat][RHEL-lifecycle]
+[SUSE Enterprise Linux (SLES)][SLES]  | 12 SP5+               | x64               | [SUSE][SLES-lifecycle]
+[Ubuntu][Ubuntu]                      | 20.04+                | x64, Arm64, Arm32 | [Ubuntu][Ubuntu-lifecycle]
+
+Other distributions are supported at best effort, per [.NET Support and Compatibility for Linux Distributions](../../linux-support.md).
+
+### Libc compatibility
+
+- [glibc][glibc] 2.23 (from Ubuntu 16.04)
```
Comment (jkotas) [inline release-notes/9.0/supported-os.md:43]: The discussion in https://github.com/dotnet/runtime/issues/91826 suggested to bump this to 18.04. Is it intentional that we are keeping this at 16.04?

cc @richlander

Comment (richlander) [reply release-notes/9.0/supported-os.md:43]: You are right. 18.04.

Comment (richlander) [reply release-notes/9.0/supported-os.md:43]:
```suggestion
- [glibc][glibc] 2.27 (from Ubuntu 18.04)
```

File: release-notes/9.0/supported-os.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+OS                                    | Version                 | Architectures   | Lifecycle
+--------------------------------------|-------------------------|-----------------|----------
+[Windows 10 Client][Windows-client]   | Version 1607+           | x64, x86, Arm64 | [Windows][Windows-lifecycle]
+[Windows 11][Windows-client]          | Version 22000+          | x64, x86, Arm64 | [Windows][Windows-lifecycle]
+[Windows Server][Windows-Server]      | 2012+                   | x64, x86        | [Windows Server][Windows-Server-lifecycle]
+[Windows Server Core][Windows-Server] | 2012+                   | x64, x86        | [Windows Server][Windows-Server-lifecycle]
+[Nano Server][Nano-Server]            | Version 1809+           | x64             | [Windows Server][Windows-Server-lifecycle]
+
+[Windows-client]: https://www.microsoft.com/windows/
```
Comment (richlander) [inline release-notes/9.0/supported-os.md:17]: Is there a better link than this? It is very salesy.

Comment (akoeplinger) [reply release-notes/9.0/supported-os.md:17]: there's an MS Learn one similar to what we use for Windows Server which looks better to me and doesn't fail the markdownlint check: https://github.com/dotnet/core/pull/9145

**Pull request merged**